paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -106,7 +106,77 @@ internal class CustomerSheetTest {
     }
 
     @Test
-    fun testCardNotSavedOnConnfirmError() = activityScenarioRule.runCustomerSheetTest(
+    fun testSuccessfulCardSaveWithCardBrandChoice() = activityScenarioRule.runCustomerSheetTest(
+        integrationType = integrationType,
+        resultCallback = { result ->
+            assert(result is CustomerSheetResult.Selected)
```

Comment:
Can we change this to an `assertThat` instead? Looks like I missed this in the last PR, if you don't mind changing both occurrences of it.